src/main/resources/data/pokecube_mobs/database/pokemobs/spawns/plains.json

```diff
@@ -145,7 +145,7 @@
       ]
     },
     {
-      "and_preset": "savannas, lakes",
+      "and_preset": "savannas, rivers",
```

Comment:
Can you confirm that this will work? rivers used to not be able to count as other biomes

Reply:
this should've been removed with the initial commit

Reply:
it appears to be in the pull request however, so you may need to see about changing the file back?